setup.sh

```diff
@@ -195,8 +189,23 @@ main() {
     install_grafana || :
   fi
 
+  if [ -n "$install_logging" ]; then
+    echo "> Installing Elasticsearch..."
+    install_elasticsearch || :
+
+    echo "> Installing Fluentd..."
+    install_fluentd || :
+
+    echo "> Installing Kibana..."
+    install_kibana || :
+  fi
+
   echo "> Installing Ververica Platform..."
-  install_vvp "$edition" "$install_metrics" || :
+  install_vvp "$edition" "$install_metrics" "$install_logging" || :
+
+  echo "> Waiting for all Deployments and Pods to become ready..."
+  kubectl --namespace vvp wait --timeout=5m --for=condition=available deployments --all
```

Comment:
I think, there is "--wait" parameter in helm to achieve this.

Reply:
yes, there is, but that one only works one-by-one...so installing would not not overlap and may actually take more time, unfortunately